guides/doc-Content_Management_Guide/topics/Importing_Content.adoc

```diff
@@ -451,8 +451,11 @@ Use this option if you have one of the following errors:
 . In the {Project} web UI, navigate to *Content* > *Products*.
 . Select the product containing the corrupted repository.
 . Select the name of a repository you want to synchronize.
-. From the *Select Action* menu, select *Advanced Sync*.
-. Select the option and click *Sync*.
+. To verify the checksum, click *Verify Content Checksum* from the *Action* menu.
+. To perform optimised sync and complete sync, select *Advanced Sync*, perform the following:
```

Comment:
We don't use subsections and I don't see the value of a subsection here as this is as clear when written in a linear procedure. Are you performing one or two syncs? If you are selecting one of two, you need to use or rather than and. 
```suggestion
. To perform optimised sync or complete sync, select *Advanced Sync*.
```